Assets/HoloToolkit-Examples/ColorPicker/ColorPickerExample.unity

```diff
@@ -188,7 +188,7 @@ Canvas:
   m_Enabled: 1
   serializedVersion: 3
   m_RenderMode: 2
-  m_Camera: {fileID: 0}
+  m_Camera: {fileID: 1596720591}
```

Comment:
Why does the PR show changes to files from the merge?

Reply:
No idea. Could be Unity touching the files as they were updated.

Reply:
No, then the changes would show in the meta files.

It looks like the commits have been duplicated some how. Please fix so the PR only shows the changes you're planning to commit.